third_party/move/move-vm/runtime/src/runtime_type_checks.rs

```diff
@@ -455,25 +443,8 @@ impl RuntimeTypeCheck for FullRuntimeTypeCheck {
                 let field_ref_ty = ty_builder.create_ref_ty(field_ty, is_mut)?;
                 operand_stack.push_ty(field_ref_ty)?;
             },
-            Bytecode::PackClosure(fh_idx, mask) => {
-                let function = frame.build_loaded_function_from_handle_and_ty_args(
-                    module_storage,
-                    *fh_idx,
-                    vec![],
-                )?;
-                verify_pack_closure(ty_builder, operand_stack, &function, *mask)?;
-            },
-            Bytecode::PackClosureGeneric(fh_idx, mask) => {
-                let ty_args = frame.instantiate_generic_function(
-                    Option::<&mut UnmeteredGasMeter>::None, // don't charge for paranoid mode
-                    *fh_idx,
-                )?;
-                let function = frame.build_loaded_function_from_instantiation_and_ty_args(
-                    module_storage,
-                    *fh_idx,
-                    ty_args,
-                )?;
-                verify_pack_closure(ty_builder, operand_stack, &function, *mask)?;
+            Bytecode::PackClosure(..) | Bytecode::PackClosureGeneric(..) => {
+                // Skip: runtime checks are implemented in interpreter loop!
```

Comment:
where is this now?

Reply:
Technically I can leave it as is, but then for lazy loading I need to charge gas for load, so I would end up passing `Option<&mut impl GasMeter>` to `build_loaded_function_from_handle_and_ty_args` + transaction context, which is rather ugly.  Here, we charge at load, and then do the paranoid check

Reply:
I missed it somehow, great